tts/Object.def.lua

```diff
@@ -397,6 +399,24 @@ function Container.getObjects() end
 ---@return tts__Object[]
 function ScriptingTrigger.getObjects() end
 
+--- Places an object into a container.
+---
+---@param object tts__Object
+---@return self
+function Container.putObject(object) end
+
+--- Places an chip onto another chip, forming a stack.
+---
+---@param object tts__Stackable
+---@return self
+function Stackable.putObject(object) end
+
+--- Places an card onto another card, forming a deck.
+---
+---@param object tts__Card
+---@return tts__Deck
+function Card.putObject(object) end
```

Comment:
not sure how this works with the inheritance on cardcustom/deckcustom. does there need to be another definition for those?